projects/storefrontlib/shared/components/form/form-errors/form-errors.component.spec.ts

```diff
@@ -104,6 +104,24 @@ describe('FormErrors', () => {
     );
   });
 
+  describe('a11y', () => {
+    it('should have `aria` attributes', () => {
+      fixture.detectChanges();
+      const ariaLive =
```

Comment:
I would recommend the `aria-live` attribute is the only thing we need to test because its value is conditional. We should check the value is correct depending on the condition (ie. ariaLiveToken).